server/services/GithubService.js

```diff
@@ -1,6 +1,21 @@
 const axios = require('axios');
+const { User } = require('../models');
 const { AuthorizationError } = require('../errors/auth');
 
+const {
+    GITHUB_GRAPHQL_SERVER,
+    GITHUB_OAUTH_SERVER,
+    GITHUB_CLIENT_ID,
+    GITHUB_CLIENT_SECRET,
+    GITHUB_TEAM_TESTER,
+    GITHUB_TEAM_ADMIN,
+    GITHUB_TEAM_ORGANIZATION,
+    GITHUB_TEAM_QUERY,
+    GITHUB_REPO_OWNER,
+    GITHUB_REPO_NAME,
+    GITHUB_REPO_ID
+} = process.env;
```

Comment:
nitpick: I had thought about it before but it may be beneficial to just control all the `process.env` values from some central config location. This helps in cases where reused `process.env` values need to be defaulted or for just making it easier to pass around said config objects, etc

Reply:
I'm open to it in the future! My initial reaction is that it would be hard to prevent people from using process.env but I'm definitely curious to see what you have in mind.